### PR TITLE
Fix execute script null and undefined argument

### PR DESCRIPTION
**Pull request by LukoyanovE**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

It's not allowed by current implementation of webdriver to pass `null` and `undefined` as arguments to `execute` and `executeAsync` methods.

### W3C Webdriver Protocol
But, if we look into [W3X webdriver specification](https://w3c.github.io/webdriver/#dfn-extract-the-script-arguments-from-a-request):
> 5. Let arguments be the result of calling the [JSON deserialize](https://w3c.github.io/webdriver/#dfn-json-deserialize) algorithm with arguments args.

Then:

> 3. Matching on value:
> - undefined
> - null
> - type Boolean
> - type Number
> - type String
> 
> Return success with data value.

There are one more detail:
instance of Array transforms by cloning,
then internal JSON clone algorithm transforms `undefined` to `null`.

Transform of `undefined` to `null` works fine.

### JSON Wire Protocol
For [jsonWP](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#WebElement_JSON_Object.md)

> Arguments may be any JSON-primitive, array, or JSON object. ...
If we look into [JSON specification](https://json.org/), we see that `null` is on of the primitives. So it should be supported too.

There are nothing about `undefined`.
But, I think, it should be transformed to `null` as w3c does.
It works too.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/webdriver/protocol/webdriver.json [hunk 1]
```diff
@@ -662,7 +662,7 @@
         "required": true
       }, {
         "name": "args",
-        "type": "(string|object|number|boolean)[]",
+        "type": "(string|object|number|boolean|null|undefined)[]",
```
Comment (christian-bromann) [inline packages/webdriver/protocol/webdriver.json]: we do a `typeof` check and `typeof null === "object"` so I think we can remove `null` here.

File: packages/webdriver/protocol/webdriver.json [hunk 2]
```diff
@@ -639,7 +639,7 @@
...
```
Comment (christian-bromann) [inline packages/webdriver/protocol/webdriver.json]: same

File: packages/webdriver/protocol/jsonwp.json [hunk 3]
```diff
@@ -721,7 +721,7 @@
         "required": true
       }, {
         "name": "args",
-        "type": "(string|object|number|boolean)[]",
+        "type": "(string|object|number|boolean|null|undefined)[]",
```
Comment (christian-bromann) [inline packages/webdriver/protocol/jsonwp.json]: same

File: packages/webdriver/protocol/jsonwp.json [hunk 4]
```diff
@@ -703,7 +703,7 @@
...
```
Comment (christian-bromann) [inline packages/webdriver/protocol/jsonwp.json]: same

**Comment by christian-bromann** [comment]
Also could you please sign the CLA?

**Comment by LukoyanovE** [comment]
> Also could you please sign the CLA?

done

**Comment by christian-bromann** [comment]
the cla hook is still pending, did you sign with the same email you did the commit with?

**Comment by LukoyanovE** [comment]
> the cla hook is still pending, did you sign with the same email you did the commit with?

looked on config, nope - different email

**Comment by christian-bromann** [comment]
> looked on config, nope - different email

What do you mean?

**Pull request closed without merging**